views/homepage.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Kainos</title>
+    <link rel="icon" href="/images/favicon.png">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<style>
+    div.nav {
+        margin-bottom: 0;
+        border-radius: 0;
+    }
+
+    /* Modify the background color */
+    div.nav {
+        background-color: rgb(40, 53, 131);
+        font-family: "Galano Grotesque";
+        color: aliceblue;
+    }
+
+    body {
+        font-family: "Galano Grotesque";
+        font-size: 1.6rem;
+        letter-spacing: 0;
+        line-height: 2.4rem;
+        margin: 0;
+    }
+
+    footer {
+        background-color: rgb(40, 53, 131);
+        font-family: "Galano Grotesque";
+        font-size: 1.6rem;
+        color: white;
+        padding-top: 10px;
+        height: 300px;
+        width: 100%;
+        position: fixed;
+        bottom: 0;
+        text-align: center;
+    }
+
+    .Belfast-contact {
+        float: left;
+        padding-left: 100px;
+    }
+
+    .Gdansk-contact {
+        float: right;
+        padding-right: 100px;
+
+    }
+</style>
+
+<body class="body">
+    <div class="nav">
```

Comment:
move this section to a heading.html file and import. that way we can reuse on other pages